src/HotChocolate/Neo4J/src/Data/Paging/Neo4JOffsetPagingProvider.cs

```diff
@@ -81,21 +76,31 @@ private Neo4JExecutable<TEntity> CreatePagingContainer(object source)
             Neo4JExecutable<TEntity> queryable,
             OffsetPagingArguments arguments = default)
         {
-            if (arguments.Skip.HasValue)
+            if (arguments.Take.HasValue)
```

Comment:
Regarding the comment from above. We do know what values we have set on the queryable WithLimit and WithSkip. so in this cope we actually know if we have to slice  -1